packages/esm-patient-immunizations-app/package.json

```diff
@@ -37,13 +37,15 @@
     "url": "https://github.com/openmrs/openmrs-esm-patient-chart/issues"
   },
   "dependencies": {
-    "@carbon/react": "^1.12.0",
-    "@openmrs/esm-patient-common-lib": "^6.1.0",
-    "lodash-es": "^4.17.21"
+    "@hookform/resolvers": "^3.3.1",
+    "lodash-es": "^4.17.21",
+    "react-hook-form": "^7.46.2",
+    "zod": "^3.22.2"
   },
   "peerDependencies": {
+    "@carbon/react": "1.x",
```

Comment:
`@carbon/react` should definitely be a dependency and not a peer dependency.

Reply:
I found myself in a bit of confusion when deciding whether to designate Carbon as a peer or a direct dependency, primarily due to inconsistencies observed across the repositories. For instance, I'm curious if its inclusion as a peer in the [patient-chart-app package](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-chart-app/package.json#L44) is a residual choice from a past configuration related to the patient-chart-app package.

Reply:
I think the patient-chart-app is in error or at least, its inconsistent with how we handle this in most apps. Actually now that I look at it in detail, we probably should move towards marking this as a peerDependency, but that should be an ecosystem-wide decision rather than a one-off we're making in individual apps.

Reply:
For now, I moved it under "dependencies"